Source/Lib/Codec/EbEntropyCoding.c

```diff
@@ -5343,6 +5343,28 @@ static void CodeProfileTierLevel(
 	return;
 }
 
+/**************************************************
+* DetermineMax
+**************************************************/
+EB_U32 DetermineNumReorderPics(SequenceControlSet_t    *scsPtr)
```

Comment:
Recommend: DetermineNumReorderPics ==> ComputeNumReorderPics, which seems the convention in SVT.
Also internal CI stopped. Pls run CI fast with this changes locally(Note, needs install libs), to check if the HM decoder runs fine.

Reply:
CI Fast has been run locally without problems.  Also encoded files have been viewed in VQ Analyzer to confirm they can be decoded and SPS/VPS value for MaxNumReorderPics is appropriate.